### DISCUSSION TITLE
GPU not working localy on Ubuntu

### DISCUSSION THREAD

**Discussion by nguyenanhtuan1008** [pieces-app/support#91] [Q&A]
I'm using ubuntu but seems gpu not working on localy. 
What should I do? 
Do I miss any installing step, I just follow the **Linux** installation tutorial in homepage.
![Screenshot from 2024-02-16 13-41-53](https://github.com/pieces-app/support/assets/36155295/9b558df2-6567-4cd0-8de7-b2941ccb6a7d)

**Comment by caleb-at-pieces** [comment]
This might do better as an issue on this repo, taggin @brian-pieces as he is the one with the most knowledge about this on our team.

**Comment by brian-at-pieces** [comment]
Hey @nguyenanhtuan1008 sorry you're having an issue. Would you mind running the `vulkaninfo` command in your terminal and reporting the results? Thanks!

**Comment by rosie-at-pieces** [comment]
closing this as it is stale. @nguyenanhtuan1008 if you are still having an issue with this please [open an issue](https://github.com/pieces-app/support/issues/new/choose).
